### PR TITLE
[Minor Changes]: Add a link for resources and fixed styling issues

### PR DESCRIPTION
**Pull request by NachtSpyder04**
Changes from this PR:
- about page will have link for the resources 
- fixed styling issues of the title when viewed  in mobile view

image
image

### CODE REVIEW

**Comment by aPR0T0** [comment]
image
Image seems squashed, just give width and don't specify height or set it to auto, will help.

File: _layouts/about.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
            {% if site.title == "blank" -%}<span class="font-weight-bold">{{ site.first_name }}</span> {{ site.middle_name }} {{ site.last_name }}{%- else -%}{{ site.title }}{%- endif %}
           </h1> -->
           <!-- add image for heading -->
-          <img src="{{ site.baseurl }}/assets/img/EKLAVYA_2024__6_-removebg-preview.png" alt="heading" style="width: 500px; height: 100px; object-fit: cover; object-position: 50% 25%;">
+          <img src="{{ site.baseurl }}/assets/img/EKLAVYA_2024__6_-removebg-preview.png" alt="heading" style="width: 500px; height: 100px; object-fit: fill; object-position: 50% 25%;">
```
Comment (aPR0T0) [inline _layouts/about.html:12]: If you won't specify height, it will maintain the ratio only

**Review by aPR0T0 — APPROVED** [review]

File: _layouts/about.html (hunk 1 above)
Comment (NachtSpyder04) [reply _layouts/about.html:12]: image
Is this fine, I have set it to auto

Comment (aPR0T0) [reply _layouts/about.html:12]: yeah looks gr8

**Pull request merged**